README.md

```diff
@@ -36,7 +36,7 @@ You only need to install Docker, nothing else is required.
 
 ### Manual Installation for Development
 
-For development, we recommend to clone the repository and install AMIV API
+For development, we recommend to **fork** the repository and install AMIV API
```

Comment:
Nitpick: I don't think we need to highlight this word. I would remove the stars.